pom.xml

```diff
@@ -12,12 +12,12 @@
     <properties>
         <zipkin-logzio-version>0.0.5</zipkin-logzio-version>
         <!-- make sure this matches zipkin's version -->
-        <zipkin.version>2.16.0</zipkin.version>
```

Comment:
lens was available last august when this version was released, but it was optional (you'd have to click Try Lens) the current version it is default